apps/synapse-portal-framework/src/style/components/csbc-home-page/_ecosystem.scss

```diff
@@ -17,14 +17,9 @@
     margin-bottom: 20px;
   }
   .control-container {
-    display: flex;
-    height: 450px;
```

Comment:
Updated the component to 
1. Use MUI Tab components, so some of these styles were no longer necessary
2. Use MUI breakpoints, so some styles were moved to the component itself